src/frontend/src/components/DesignReviewDelayModal.tsx

```diff
@@ -0,0 +1,58 @@
+import { TextField, Button, Link } from '@mui/material';
+import { useState, ChangeEvent } from 'react';
+import { DesignReview, wbsPipe } from 'shared';
+import { useToast } from '../hooks/toasts.hooks';
+import { routes } from '../utils/routes';
+import NERModal from './NERModal';
+import { Link as RouterLink } from 'react-router-dom';
+
+// delay wp modal component
+export const DesignReviewDelayModal: React.FC<{ open: boolean; onHide: () => void; dr: DesignReview }> = ({
+  open,
+  onHide,
+  dr
+}) => {
+  const toast = useToast();
+  const [weeks, setWeeks] = useState<number>(1);
+  const onChange = (e: ChangeEvent<HTMLInputElement>) => {
+    if (e.target.value === '' || parseInt(e.target.value) >= 1) {
+      setWeeks(parseInt(e.target.value));
+    } else {
+      toast.error('If delaying, it must be by at least 1 week');
+    }
+  };
+
+  return (
+    <NERModal open={open} title="Delay WP" onHide={onHide} hideFormButtons showCloseButton>
+      <TextField
+        type="number"
+        label="Enter number of weeks"
```

Comment:
this should be in a FormLabel right above this Textfield instead of in this label, see how we do it in pretty much any other form